server/instance.go

```diff
@@ -149,6 +149,21 @@ func (s *Server) registerInstanceRoutes(g *echo.Group) {
 					return err
 				}
 			}
+			// Ensure all databases belong to this instance are under the default project before instance is archived.
+			if v := instancePatch.RowStatus; v != nil && *v == api.Archived.String() {
+				defaultProjectID := 1
+				databases, err := s.store.FindDatabase(ctx, &api.DatabaseFind{
+					NotProjectID: &defaultProjectID,
```

Comment:
Instead of adding this where clause, can we just find all databases in an instance and show the ones that are not in the default project?

Reply:
But then we still need this where clause to filter

Reply:
For-loop in code will just do the job. We cannot add thousands of clauses in the API.